src/components/Login/LoginContainer.test.tsx

```diff
@@ -77,11 +78,13 @@ describe("LoginContainer component tests", () => {
     cleanup();
   });
 
-  it("should render LoginScanCard", () => {
+  it("should render LoginScanCard", async () => {
```

Comment:
since this screen is blocked, all tests require additional button click to reach original screen